pkg/collector/corechecks/oracle-dbm/statements.go

```diff
@@ -539,7 +541,7 @@ func (c *Check) StatementMetrics() (int, error) {
 		log.Tracef("number of collected metrics with SQL_ID %+v", len(statementMetrics))
 		statementMetricsAll = append(statementMetricsAll, statementMetrics...)
 		SQLCount = len(statementMetricsAll)
-		sender.Count("dd.oracle.statements_metrics.sql_count", float64(SQLCount), "", c.tags)
+		//sender.Count("dd.oracle.statements_metrics.sql_count", float64(SQLCount), "", c.tags)
```

Comment:
Curious why comment out this metric?

Reply:
> Curious why comment out this metric?

It hasn't proved useful so far. But if you think we should keep it, I can remove the comment again.

Reply:
I don't have enough historical context to say we should keep it or not. Asked because I feel this line of code does not contribute to the memory leak issue. So curious to understand the reason of removing it. But your call to keep or remove it 😄